ssh/browser/browser.go

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Builds and serves the static site that is generated for the browser/Wasm
```

Comment:
// The browser command ...

https://github.com/golang/go/wiki/CodeReviewComments#package-comments